src/simularium/RemoteSimulator.ts

```diff
@@ -120,244 +65,124 @@ export class RemoteSimulator implements ISimulator {
         this.onTrajectoryDataArrive = handler;
     }
 
-    /**
-     * WebSocket State
-     */
-    private socketIsConnecting(): boolean {
-        return (
-            this.webSocket !== null &&
-            this.webSocket.readyState === this.webSocket.CONNECTING
-        );
-    }
-
     public socketIsValid(): boolean {
-        return !(
-            this.webSocket === null ||
-            this.webSocket.readyState === this.webSocket.CLOSED
-        );
-    }
-
-    private socketIsConnected(): boolean {
-        return (
-            this.webSocket !== null &&
-            this.webSocket.readyState === this.webSocket.OPEN
-        );
+        return this.webSocketClient.socketIsValid();
```

Comment:
who calls socketIsValid? could we remove this?

Reply:
it's called in `configureNetwork()` and `disableNetworkCommands()` inside `SimulariumController`. This was mostly me just trying to maintain the existing functionality from before as I refactored everything, I don't 100% understand everything that the `SimulariumController` does so I can't tell how important it is 😬